core/src/main/java/org/infinispan/interceptors/distribution/VersionedDistributionInterceptor.java

```diff
@@ -26,6 +28,13 @@
 
    private static final Log log = LogFactory.getLog(VersionedDistributionInterceptor.class);
 
+   private RpcOptions rpcOptions;
+
+   @Start
+   public void setRpcOptions() {
+      rpcOptions = rpcManager.getRpcOptionsBuilder(ResponseMode.SYNCHRONOUS_IGNORE_LEAVERS, DeliverOrder.NONE).build();
```

Comment:
I think the prepare RPC options can moved to `TxDistributionInterceptor`, because the `prepareOnAffectedNodes` sync parameter is always a field (and it's always `true` when write skew is enabled).